src/verifiers/documentStatus/didSigned/didSignedDocumentStatus.test.ts

```diff
@@ -624,12 +638,12 @@ describe("verify", () => {
       const res = await openAttestationDidSignedDocumentStatus.verify(docWithIncorrectRevocation as any, options);
       expect(res).toMatchInlineSnapshot(`
         Object {
-          "data": [Error: revocation type not found for an issuer],
+          "data": [Error: Document does not match either v2 or v3 formats],
           "name": "OpenAttestationDidSignedDocumentStatus",
           "reason": Object {
-            "code": 9,
-            "codeString": "UNRECOGNIZED_REVOCATION_TYPE",
-            "message": "revocation type not found for an issuer",
+            "code": 8,
+            "codeString": "UNRECOGNIZED_DOCUMENT",
+            "message": "Document does not match either v2 or v3 formats",
```

Comment:
feels like the error message became less useful

Reply:
It's less useful I agree but fixing that would be a bit troublesome. Is there any huge concern ?